core/schema/container.graphqls

```diff
@@ -23,27 +24,39 @@ type Container {
   "Creates a named sub-pipeline"
   pipeline(name: String!, description: String): Container!
 
-  "Initializes this container from the base image published at the given address."
+  """
+  Initializes this container from the base image published at the given address.
+  Example:
+  from("docker.io/dagger/dagger:main")
+  """
   from(
     """
     Image's address from its registry.
-    Formatted as {host}/{user}/{repo}:{tag} (e.g. docker.io/dagger/dagger:main).
+    Formatted as {host}/{user}/{repo}:{tag}.
     """
     address: String!
   ): Container!
 
-  "Initializes this container from a Dockerfile build, using the context, a dockerfile file path and some additional buildArgs."
+  """
+  Initializes this container from a Dockerfile build, using the context, a dockerfile file path and some additional buildArgs.
+  Example:
+  build("directory-id")
+  build("directory-id", { dockerfile: "./frontend.Dockerfile" })
+  build("directory-id", { buildArgs: [{ name: "MODE", value: "production }] })
```

Comment:
These examples are specific to the NodeJS SDK, they don't work even for GraphQL.

Besides, the `"directory-id"` value is misleading as a proper value for a `DirectoryID`.

Reply:
It's a function call example yeah, it's abstract.

Reply:
You can also use a variable (e.g., `$directoryID`) or a truncated —but more correct— value (e.g., `"eyJmc...9fQ=="`).